i18n/en.json

```diff
@@ -12,7 +12,7 @@
+	"createwiki-defaultmainpage": "__NOTOC__\n== Welcome to {{SITENAME}}! ==\nThis Main Page was created automatically upon wiki creation and it seems it hasn't been replaced yet. The bureaucrat(s) might still be working on a Main Page, so please check again later!\n\n=== For the bureaucrat(s) of this wiki ===\nHello and welcome to your new wiki! Thank you for choosing Miraheze to host your wiki; we hope you will enjoy our hosting. You can start working on your wiki whenever you want.\n\nNeed help? No problem! To start, here are some resources we encourage you to review or skim:\n* [[mw:Special:MyLanguage/Help:Contents|MediaWiki guide]] for wiki basics: navigation, editing, general features of the software\n* [[m:Special:MyLanguage/Infoboxes|Infoboxes]] for the various ways you can get these on your wiki. PortableInfoboxes is a common choice if you're not sure where to start\n* [[mh:meta:Special:MyLanguage/ManageWiki|ManageWiki]] for further customization of the wiki's settings (favicon, extensions, permissions and more). Read carefully: certain changes can be difficult to reverse\n* [[mh:meta:Special:MyLanguage/FAQ|Miraheze FAQ]] for a general overview of Miraheze\n* [[mh:dev|Miraheze Developers Wiki]], a resource for everyone (yes you!) to find templates (infoboxes, citations...) styling, and other useful accessories to spruce up the wiki. You may find shared images at [[mh:commons|Miraheze Commons]] and [[c:|Wikimedia Commons]], and try out administrator permissions on our [[mh:test|Public Test Wiki]] as well.\n\nAdditionally if you're not sure how to make a new page, you have a few options: make a link on an existing page and follow it, edit the address bar to go to the page you want, or go to [[Special:ManageWiki/extensions]] and enable [[mw:Extension:CreatePageUw|CreatePageUw]] to add a handy link in the sidebar!\n\n==== I still don't understand ''X''! ====\nNo problem! Not everything may be in the FAQs or sufficiently clear, and we're happy to help! You can find us here:\n* On [[m:Special:MyLanguage/Help center|Miraheze Meta's Help Center]] or [[m:Community portal|]]\n* On [https://miraheze.org/discord Discord]\n* On IRC in {{Libera|miraheze}} on Libera.Chat\n* On [[phorge:|our issue tracker]] if you encounter MediaWiki errors messages, or want to request new extensions or configuration changes not already available in ManageWiki\n",
```

Comment:
This should be it's own PR.

Reply:
This was it's own PR that I closed per feedback as this will be moved to loadouts probably. I just accidentally left the commit on the same branch(I thought i discarded it). Erm, sorry 😅